<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QPushButton" name="generalDefaults">
         <property name="text">
          <string>Defaults</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="saveHistoryBox">
             <property name="text">
              <string>Save history</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearHistoryBtn">
             <property name="text">
              <string>Clear now!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="historyDetailsGroup">
           <property name="title">
            <string>History details</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="mainPageRadio">
              <property name="text">
               <string>Main pages only</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="allGemsRadio">
              <property name="text">
               <string>All gems visited</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="historyTimingGroup">
           <property name="title">
            <string>Timing</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="foreverBtn">
              <property name="text">
               <string>Forever</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="daysBtn">
              <property name="text">
               <string>Days:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="historyDays"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Caching</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QCheckBox" name="cacheGems">
              <property name="text">
               <string>Cache gems</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cacheImages">
              <property name="text">
               <string>Cache images</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Current cache size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="cacheSize">
              <property name="minimumSize">
               <size>
                <width>71</width>
                <height>22</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cacheUnits">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="clearCacheBtn">
            <property name="text">
             <string>Clear cache</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Visual</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="styleItems">
           <item>
            <property name="text">
             <string>Plain text</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Links</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Quotations</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Heading 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Heading 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Heading 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Preformatted </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lists</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="visualDefaults">
           <property name="text">
            <string>Restore defaults</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="overrideBox">
           <property name="text">
            <string>Override defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="itemOptionsGroup">
           <property name="title">
            <string>Item options</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Font family:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="fontFamilyBox"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Font:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="fontBox"/>
              </item>
              <item>
               <widget class="QToolButton" name="fontPicker">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="preferences-desktop-locale"/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Font size (pt):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="fontSize">
                <property name="minimum">
                 <number>8</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Text background:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="textBgLine">
                <property name="placeholderText">
                 <string>#aabbcc</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="txtBgPicker">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="applications-graphics"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Intendation (px):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="intednationBox">
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Text colour:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="textClrLine">
                <property name="placeholderText">
                 <string>#aabbcc</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="txtColorPicker">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="accessories-character-map"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="BoldBox">
                <property name="text">
                 <string>Bold</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ItalicBox">
                <property name="text">
                 <string>Italic</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="underlineBox">
                <property name="text">
                 <string>Underline</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>TLS</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="Browsing">
      <attribute name="title">
       <string>Browsing</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Gemini links</string>
       </property>
       <widget class="QRadioButton" name="geminiNewTab">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Open in new tab</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="geminiInPlace">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Follow in place</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Other protocols</string>
       </property>
       <widget class="QRadioButton" name="ignoreHttp">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="systemHttp">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Use system browser</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="customHttp">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>651</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Start page (leave empty for nothing)</string>
       </property>
       <widget class="QLineEdit" name="startPage">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>621</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>621</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>gemini://</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="HelpBtn">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="defaultsBtn">
       <property name="text">
        <string>Restore all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
